### PR TITLE
fix: performance headers

### LINKED ISSUE #1070: SentryTransaction.h can't be found?
When I try to integrate `7.0.0-beta.0` this way in `Package.swift` (from a Package itself):

```
dependencies: [
        .package(name: "Sentry", url: "https://github.com/getsentry/sentry-cocoa.git", from: "7.0.0-beta.0"),
(...)
    targets: [
        .target(
            name: "MyApp",
            dependencies: [
                "Sentry",
(...)
```
This is what I've got...

`~/Library/Developer/Xcode/DerivedData/MyApp-dkvqkhmiuwguoehaikzxuapvpjsv/SourcePackages/checkouts/sentry-cocoa/Sources/Sentry/SentryTransaction.m:1:9: In file included from /Users/mc/Library/Developer/Xcode/DerivedData/MyApp-dkvqkhmiuwguoehaikzxuapvpjsv/SourcePackages/checkouts/sentry-cocoa/Sources/Sentry/SentryTransaction.m:1:`

### PR DESCRIPTION
**Pull request by bruno-garcia**
Resolves #1070

We definitely need a step in CI to validate this going forward.

### CODE REVIEW

**Review by brustolin — APPROVED** [review]
LGTM ;)

**Comment by bruno-garcia** [comment]
CI stuck all the time on this repo, I wonder why

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - fix: SentryOptions.tracesSampleRate default value (#1069)
 - ref: Discard unfinished spans when capturing transaction (#1066)
 - ref: Make calls to customSamplingContext nonnull (#1061)
+- fix: fix: performance headers (#1079)
```
Comment (philipphofmann) [inline CHANGELOG.md:8]: 👀

Comment (philipphofmann) [reply CHANGELOG.md:8]: Fixed with https://github.com/getsentry/sentry-cocoa/commit/e14fa827cc83c0cb0e0dd9f2257d510654c000ea

Comment (bruno-garcia) [reply CHANGELOG.md:8]: I had to read it twice to notice it

**Pull request merged**